cmd/api/src/api/v2/auth/auth.go

```diff
@@ -139,6 +139,9 @@ func (s ManagementResource) GetSAMLProvider(response http.ResponseWriter, reques
 	} else if provider, err := s.db.GetSAMLProvider(request.Context(), int32(providerID)); err != nil {
 		api.HandleDatabaseError(request, response, err)
 	} else {
+		// Format the service provider uri's in response
+		provider = bhsaml.FormatSAMLProviderURLs(request.Context(), provider)[0]
```

Comment:
Yes this is hideously ugly, it is refactored into a nice and clean method in the future
![image](https://media1.tenor.com/m/pljAqxYDG0cAAAAC/future-squidward.gif)